scripts/esbuild-plugins/utils.js

```diff
@@ -79,9 +79,10 @@ exports.createAliasReplacer = (aliases) => {
  * @returns {RegExp}
  */
 exports.createFilterRegExpFromAliases = (aliases) => {
-  if (aliases.some(alias => alias.includes('/'))) {
-    throw new Error('Path aliases may not include slash characters.')
-  }
+  // TODO: why was this added?
```

Comment:
I'm guessing probably to avoid path traversal attacks? Although in that case a whitelist of characters might be better. There's `\\` (double backslash) which is a standard path separator, and single backslash will work on Windows as well.

Reply:
I didn't allow slashes in path aliases because otherwise it would be more difficult to implement correctly, in particular regarding ordering of the given alias map. I wanted to make sure the given ordering would not be significant.

Suppose we have the following alias entries:
```js
{
  '@views': './frontend/views',
  '@views/utils': './frontend/views/utils'
}
```
The current code would build an incorrect regex in this case, one that would first match `@views` rather than `@views/utils` upon encoutering a path like `@views/utils/misc.js`.

Fixing that kind of edge cases would require either enforcing correct key ordering in the alias map, or more complex code.